### ISSUE TITLE
Ready-to-use binary

### ISSUE THREAD

**Issue by konradorlinski** [biemster/st17h66_FindMy#8]
Hi!

The project looks really promising, and I would love to test it out, but I don't have experience with compiling these types of projects. I tried installing Keil uVision, but after wasting several hours without success, I decided to ask here for a ready-to-use binary file. Would anyone be able to share the compiled firmware? I can handle the flashing process myself.

Thank you in advance for your help!

**Comment by biemster** [comment]
https://github.com/biemster/st17h66_FindMy/blob/main/FindMy/bin/FindMy.hex

**Comment by biemster** [comment]
Also have a look at this: https://github.com/biemster/st17h66_RF
although you'll need to implement the proper FindMy broadcast yourself (which is not hard, just use this repo as example), but it compiles with GCC. The plan is to make that repo the default and do away with Keil altogether, but I did not get to that yet.

**Comment by biemster** [comment]
I'm confusing myself with all the FindMy repos I made, you actually should use this: https://github.com/biemster/FindMy/tree/main/Lenze_ST17H66

**Comment by biemster** [comment]
Although that might not have been optimized for low power, have a look at issue #7 (LONG thread, but has a solution at the end)

**Comment by konradorlinski** [comment]
Thanks for your help! I admit I missed that binary file in the repository. 
I'll try uploading it and will get back to you if I have any problems.

**Comment by konradorlinski** [comment]
I tried to upload this binary using Lenze Flasher as well as your and vadimkozhin script in Python. After supposedly uploading correctly (no errors, communication with MCU proceeds correctly) nothing happens with the tag. 

My hardware: Typical iTAG eg. [Photo](https://m.media-amazon.com/images/I/512sHEs-UoL.__AC_SX300_SY300_QL70_ML2_.jpg)

Once uploaded, it should automatically be detected on the phone or do I need to do something else? Can you advise me to help?

**Comment by biemster** [comment]
The tag should then start sending out the FindMy advertisement. You could check that with a bluetooth sniffer like `nRF Connect` on your phone.

**Comment by konradorlinski** [comment]
I did some more research on my problem and it appears that I am doing everything correctly. The tag with the default key broadcasts at: D1:22:33:44:55:66, (which agrees with the source code) and when I upload it with a generated key (Advertisement key) it appears under a different one (I found this by the distance in the BLE Scanner application and disconnecting the tag's power supply). 

I'm just wondering if I'm making a wrong assumption - should a tag programmed this way appear to be added in the Find My app on the iPhone or not? I recently bought a tag from aliexpress, which fully cooperate with Find My (it appears in the Find My app on the iPhone), and hence my question about the app on the phone. The tag is based on the ST17H65B processor.


I found the answer to my question: https://github.com/seemoo-lab/openhaystack/issues/39
